examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -995,7 +999,7 @@ def compute_text_embeddings(prompt, text_encoders, tokenizers):
     # We need to initialize the trackers we use, and also store our configuration.
     # The trackers initializes automatically on the main process.
     if accelerator.is_main_process:
-        accelerator.init_trackers("dreambooth-lora-sd-xl", config=vars(args))
+        accelerator.init_trackers("dreambooth-lora-sd-xl-debug", config=vars(args))
```

Comment:
Not sure why this is not updating. As we can see this is NOT the case:

https://github.com/huggingface/diffusers/blob/1df38f345790208e72fc34df6237818608a082aa/examples/dreambooth/train_dreambooth_lora_sdxl.py#L1002